### ISSUE TITLE
mongodb_status: Issue with rs state validation

### ISSUE THREAD

**Issue by rhysmeister** [ansible-collections/community.mongodb#210]
##### SUMMARY
Spotted this in the int tests...

```
TASK [Check the status of the replicaset with mongodb_status module (no auth) - 2] ***
ok: [testhost] => {"changed": false, "iterations": 2, "msg": "replicaset is in a converged state", "replicaset": {"localhost:3001": "PRIMARY", "localhost:3002": "STARTUP2", "localhost:3003": "STARTUP2"}}
```

The module should not consider this a "good" state.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
mongodb_status

**Comment by rhysmeister** [comment]
This is the dumb bit of code...

```python
if (statuses.count("PRIMARY") == 1
                and ((statuses.count("SECONDARY")
                     + statuses.count("ARBITER")) % 2 == 0)):
```

So in our test example 1 x PRIMARY, 2 x STARTUP2...

Count of SECONDARY = 0
Count of ARBITER = 0
(0 + 0) % 2 = 0

I need to validate that only these three statuses exist in the result.

**Comment by rhysmeister** [comment]
Something based around the following should work...

good = the valid states
one, two , three, four = replicaset statuses. one and two are good, the others not

>>> good = ["PRIMARY", "SECONDARY", "ARBITER"]
>>> one = ["PRIMARY", "SECONDARY", "SECONDARY"]
>>> two = ["PRIMARY", "SECONDARY", "ARBITER"]
>>> three = ["PRIMARY", "STARTUP2", "STARTUP2" ]
>>> four = ["PRIMARY", "STARTUP2", "SECONDARY"]
>>> set(one) - set(good)
set()
>>> set(two) - set(good)
set()
>>> set(three) - set(good)
{'STARTUP2'}
>>> set(four) - set(good)
{'STARTUP2'}

**Comment by rhysmeister** [comment]
Fixed in #211
